config-bot/main

```diff
@@ -4,7 +4,7 @@ import os
 import re
 import sys
 import json
-import toml
+import tomllib
```

Comment:
Better solution:
```
try:
    import tomllib
except ImportError:
    import tomli as tomllib
```

Reply:
We don't need this try/except because we can just assume we'll be on python 3.11+ (we control the environment). What you have already looks good.

Reply:
Thank you.